src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private String name;
+    private Integer id;
+    private Integer volume;
+    private String type;
+    private Integer numberOfCylinders;
 
+    public Engine() {
+        this.name = "hybrid";
```

Comment:
it is bad practice to initialize fields in the constructor

Reply:
done